src/pages/Users/Login.js

```diff
@@ -17,23 +17,19 @@ class Login extends Component {
     }
   }
   render() {
-    let { email, password } = this.state.form.user
       return (
           <div>
             <div className="loginMain" >
               <form onSubmit={this.onSubmit}>
               <input
                 type="email"
                 name="email"
-                value={email}
```

Comment:
Why this change?  Its better to have your inputs be controlled by React, meaning that you set the value of the input explicitly as this code was doing.

Reply:
Something changed while hacking through a functioning login that I believed to be broken. It works, but it may be bad convention. Should I change?